### PR TITLE
Request permission for azure-container-registry-plugin

### PR DESCRIPTION
**Pull request by yuwzho**
# Description

Request  upload permission for Azure Container Registry Tasks Plugin
https://github.com/jenkinsci/azure-container-registry-tasks-plugin
https://issues.jenkins-ci.org/browse/HOSTING-681

# Submitter checklist for changing permissions

### Always

- [X] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [X] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [X] Make sure the file is created in `permissions/` directory
- [X] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [X] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [X] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [X] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [X] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [X] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

**Comment by batmat** [comment]
Apparently username is correct https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=yuwzho
 Did you actually log into artifactory as required?

**Comment by yuwzho** [comment]
@batmat I just logged into the artifactory but triggered build with failed results (Maybe there is some latency). And we find we just need only one service account to be added the permission, so I removed our personal accounts and keep only one.

File: permissions/plugin-azure-container-registry-tasks.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "azure-container-registry-tasks"
+github: "jenkinsci/azure-container-registry-tasks-plugin"
+paths:
+- "org/jenkins-ci/plugins/azure-container-registry-tasks"
```
Comment (daniel-beck) [inline permissions/plugin-azure-container-registry-tasks.yml]: This does not correctly represent pom.xml contents, please see this repo's readme.

CC @batmat

Comment (yuwzho) [reply permissions/plugin-azure-container-registry-tasks.yml]: Sorry, I have already updated the pom file

**Pull request merged**